packages/synapse-react-client/src/mocks/challenge/mockChallenge.ts

```diff
@@ -95,3 +101,22 @@ export const mockTeamList = (teams: ChallengeTeam[]): ListWrapper<Team> => {
     list,
   }
 }
+
+export const mockRegisteredChallengeTeams: ChallengeTeam[] = [
```

Comment:
Moved mock declaration to the `mockChallenge` file instead of the `challengeHandlers`